src/gc.c

```diff
@@ -1468,6 +1468,36 @@ int jl_gc_classify_pools(size_t sz, int *osize)
 
 // sweep phase
 
+gc_fragmentation_stat_t gc_page_fragmentation_stats[JL_GC_N_POOLS];
+
+STATIC_INLINE void gc_update_page_fragmentation_data(jl_gc_pagemeta_t *pg) JL_NOTSAFEPOINT
+{
+#ifdef GC_MEASURE_PAGE_FRAGMENTATION
```

Comment:
Where is this defined?

Reply:
gc.h.

I plan to enable it (and expose these data at the Julia level) in a separate PR.

For now I just want to get the basic functionality in.